helmfile/values/harbor.yaml.gotmpl

```diff
@@ -152,3 +152,6 @@ redis:
     nodeSelector: {{- toYaml .Values.harbor.nodeSelector | nindent 6 }}
     affinity:     {{- toYaml .Values.harbor.affinity | nindent 6 }}
     tolerations:  {{- toYaml .Values.harbor.tolerations | nindent 6 }}
+
+updateStrategy:
+  type: Recreate
```

Comment:
Does this apply to all components or just some? How about the database for example?

Reply:
Only the kubernetes deployments (not statefulsets), https://github.com/goharbor/harbor-helm/blob/master/values.yaml#L329-L330